File: C/Assignment11/Program1.c
//////////////////////////////////////////////////////////////////////////////////
//                                                                              //
// File name :   Program1.c                                                     //  
// Description : Program which accept number from user & return alphabets       //
// Author :      Pallavi Omprakash Malewar                                      //
// Date :        21/06/2025                                                     //
//                                                                              //
//////////////////////////////////////////////////////////////////////////////////


/*
Input : 5
Output : A   B   C   D   E
*/


#include<stdio.h>

void Pattern(int iNo)
{
    char ch = '\0';

    int iCount = 0;

    for(iCount = 0 ; iCount < iNo ; iCount++)
    {
        printf("%c\t" , 'A' + iCount);
    }
    printf("\n");
}

int main()
{
    int iValue = 0;

    printf("Enter Number : ");

    scanf("%d" , &iValue);

    Pattern(iValue);

    return 0;
}

File: JAVA/Assignment11/Program1.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Accept Number from user and display below pattern.              //
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 30 Jun 2026                                                                  //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////


/*

    Input  : 5
    Output : A   B   C   D   E

*/


import java.util.Scanner;

class Program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the number : ");
        int No = sobj.nextInt();

        Pattern(No);
    }

    public static void Pattern(int No)
    {
        char start = 'A';

        if(No == 0) return;
        if(No < 0) No = -No;

        while(No != 0)
        {
            
        }
    }
}
